src/private/xml_writer.cpp

```diff
@@ -10,16 +10,46 @@
 namespace adm {
   namespace xml {
 
-    /// Check if a option/flag is set
-    /**
+    namespace {
+      /// Check if a option/flag is set
+      /**
      * Checks if the option @a flag is set within @a options.
      *
      * This is equivalent to an bitwise AND followed by a conversion to bool,
      * but should improve readability.
      */
```

Comment:
comment format broken with indent (and was odd anyway...)

Reply:
fd9ce2141cb4f8c520678e74d899519abd1531a7